src/helpers/dimensions.ts

```diff
@@ -24,6 +24,10 @@ export function getDimensionTotals(bucket: VisualizationObject.IBucket): AFM.ITo
     );
 }
 
+export function getMeasures(buckets: VisualizationObject.IBucket[]): VisualizationObject.IBucket {
```

Comment:
should it be `getMeasuresBucket`? I think it's more explicit than current one.

Reply:
I think reuse this function: `findBucketByLocalIdentifier` (https://github.com/gooddata/gooddata-react-components/blob/master/src/helpers/dimensions.ts#L18)

Reply:
updated